bazel/linux/bundles.bzl

```diff
@@ -77,9 +76,9 @@ and an init script to run them as a kunit test.""",
             default = 5,
             doc = "Maximum recursive depth when expanding a list of kernel module dependencies.",
         ),
-        "_template_init": attr.label(
+        "_template_kunit": attr.label(
```

Comment:
while at it, it may be worth making this template (and _template_check?) public by removing the starting `_` so one can override them?

(up to you, probably worth a separate PR?)

Reply:
I will create a separate PR for this.

Reply:
Follow up PR: https://github.com/enfabrica/enkit/pull/609